sbnanaobj/Morgue/ParticleMatch.h

```diff
@@ -0,0 +1,19 @@
+////////////////////////////////////////////////////////////////////////
+// \file    ParticleMatch.h
+////////////////////////////////////////////////////////////////////////
+#ifndef PARTICLEMATCH_H
+#define PARTICLEMATCH_H
+
+namespace caf
+{
+  /// Match from a reconstructed track to a true particle  */
+  class ParticleMatch {
+  public:
+    ParticleMatch(); //!< Constructor
+    int G4ID; //!< ID of the particle match, taken from G4 */
```

Comment:
I wonder if it's worth stripping out all the comments (and field initializers in the constructor) to emphasize that this is dead code that only has to exist for technical reasons?

Reply:
yep that makes sense!